_source/_posts/2023-07-28-oidc_workshop.md

```diff
@@ -12,7 +12,11 @@ tweets:
 - ""
 image: blog/oidc_workshop/social.jpg
 type: awareness
+github: https://github.com/oktadev/okta-enterprise-ready-workshops/tree/oidc-workshop-complete
+changelog:
+- 2024-03-07: Add completed GH branch to frontmatter
```

Comment:
I don't think the changelog should be used here. This is for internal workings (meaning readers wouldn't care that a change was made to a property set they can't access or see)